<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>局域网聊天工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">image: url(:/images/widgetbg.png);</string>
     </property>
     <widget class="QComboBox" name="sizeComboBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>640</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
     </widget>
     <widget class="QToolButton" name="colorToolBtn_2">
      <property name="geometry">
       <rect>
        <x>1223</x>
        <y>30</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>更改字体颜色</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>640</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>640</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QToolButton" name="underlineToolBtn_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>635</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>下划线</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="italicToolBtn_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>635</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>倾斜</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="userTableWidget_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>220</y>
        <width>421</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bj.jpg);</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>主机名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP地址</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>110</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/camera.png</normaloff>:/images/camera.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QTextEdit" name="messageTextEdit_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>501</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="saveToolBtn_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>保存聊天记录</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="messageBrowser_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>501</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QToolButton" name="clearToolBtn_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>清空聊天记录</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="userNumLabel_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>在线用户：0人</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>113</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/voice.png</normaloff>:/images/voice.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="boldToolBtn_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>635</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>加粗</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="sendToolBtn_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>29</width>
        <height>28</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>传输文件</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>600</y>
        <width>931</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/scissors_23.434554973822px_1200891_easyicon.net.png</normaloff>:/images/scissors_23.434554973822px_1200891_easyicon.net.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
